CMake/FindGonum.cmake

```diff
@@ -0,0 +1,23 @@
+# The module defines the following variables:
+#   GONUM_FOUND - true if the Gonum was found
+#   GONUM - Gonum version number
+
+
+find_program(GO_EXECUTABLE go PATHS $ENV{HOME}/go ENV GOROOT GOPATH PATH_SUFFIXES bin)
```

Comment:
I think we can avoid the `find_program()` call here---if this `find_package()` is called after we've already found Go (which I think it is), then we are guaranteed that `GO_EXECUTABLE` is already set. :+1:

Reply:
Removed.